third_party/move/move-vm/paranoid-tests/tests/type_safety/struct/exists.mvir

```diff
@@ -19,15 +19,15 @@ module 0x2.A {
     }
 }
 
-//# run --signers 0x1 
```

Comment:
does this get regenerated and something changed in between? or can we ignore this file?

Reply:
I think there were spaces before, and I copy-pasted something, will adjust to not have this in the diff 👍

Reply:
done